example/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,27 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.signify.hue.reactivebleexample">
 
-    <!-- required for API 18 - 30 -->
-    <uses-permission
-            android:name="android.permission.BLUETOOTH"
-            android:maxSdkVersion="30" />
-    <uses-permission
-            android:name="android.permission.BLUETOOTH_ADMIN"
-            android:maxSdkVersion="30" />
-
-    <!-- required for API 23 - 30 -->
-    <uses-permission-sdk-23
-            android:name="android.permission.ACCESS_COARSE_LOCATION"
-            android:maxSdkVersion="30" />
-    <uses-permission-sdk-23
-            android:name="android.permission.ACCESS_FINE_LOCATION"
-            android:maxSdkVersion="30" />
-
-    <!-- API 31+ -->
-    <uses-permission android:name="android.permission.BLUETOOTH_CONNECT" />
-    <uses-permission
-            android:name="android.permission.BLUETOOTH_SCAN"
-            android:usesPermissionFlags="neverForLocation" />
```

Comment:
I don't think these permissions need to be removed! are needed for any app that uses android 12!

Reply:
This is for the example app, not the plugin itself. When the project is built, it will generate a manifest using the permissions it inherits for all its dependencies (including this plugin). This change was just to remove redundancy.

Reply:
Yes but the example app shows how to use the lib in a real-life app, if you don't add these permissions (with flutter-permission-handler for example ) permissions will never be handled, if you want to contribute will be better to add an example of how to use flutter-permission-handler with the library for example rather than remove permissions from example app!

Reply:
A real-life app and the example app will inherit these exact same permissions from the plugin so they actually don't actually need to be in the app's manifest (look in build/app/intermediates/merged_manifests after you build the example app). But I'm fine dropping these changes as fixing the plugin's manifest is more important to me anyway. Unfortunately I've realized for these changes to work, RxAndroidBle will need to similarly modify its AndroidManifest as its permissions get inherited too...